resources/qml/Cura.qml

```diff
@@ -73,6 +73,18 @@ UM.MainWindow
         // This has been fixed for QtQuick Controls 2 since the Shortcut item has a context property.
         Cura.Actions.parent = backgroundItem
         CuraApplication.purgeWindows()
+
+        if (CuraApplication.needToShowUserAgreement)
+        {
+            welcomeDialog.visible = true;
+            welcomeDialog.currentStep = 0;
```

Comment:
Should be the default, right?

Reply:
Yes, but can't be careful enough with these legal things like the user-agreement, what if someone changes the default for whatever reason?